adapters/tlsio_mbedtls.c

```diff
@@ -427,76 +466,66 @@ CONCRETE_IO_HANDLE tlsio_mbedtls_create(void* io_create_parameters)
             }
             else
             {
-                result->on_bytes_received = NULL;
```

Comment:
>result->on_bytes_received = NULL [](start = 16, length = 32)

please keep null initializations on create

Reply:
the TLS_IO_INSTANCE is created by using calloc which makes it NULL by default.